src/main/java/org/usfirst/frc4904/robot/subsystems/VelcroPlate.java

```diff
@@ -0,0 +1,48 @@
+package org.usfirst.frc4904.robot.subsystems;
+
+import org.usfirst.frc4904.robot.commands.VelcroPlateDefault;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class VelcroPlate extends Subsystem {
```

Comment:
Your current commands seem to all be using placeSolenoid and pickupSolenoid individually, rather than both. If when they are used, the solenoids don't pertain to each other at all, especially if you don't use both at once, then requiring VelcroPlate is bad practice because it interrupts the previous command running on the subsystem, when you only want to be interrupting commands on the specific solenoid used. This will work as is, but is bad design if it's used this way since if we replaced one of the pistons with a motor or something like that, the commands would get interrupted when we don't want them to. Either make the solenoids separate subsystems (can still be combined under VelcroPlate), or use both solenoids in your CommandGroups. (I think making them their own subsystems and requiring them individually would be a good move, while still keeping them under VelcroPlate)